source/_components/yandex_transport.markdown

```diff
@@ -0,0 +1,83 @@
+---
+title: "Yandex transport"
+description: "Instructions on how to setup Yandex transport with Home Assistant."
+logo: yandex.png
+ha_category:
+  - Sensor
+  - Transport
+ha_release: "0.100"
+---
+
+The `yandex_tranport` sensor platform uses [Yandex Maps](https://maps.yandex.ru/) it will give you the time until the next departure time from a bus/tramway/etc stop.
+
+The [Yandex Maps](https://maps.yandex.ru/) website can help to determine the id of your bus stop. You can select bus stop by clicking on the map, and look to the url:
+
+https://yandex.ru/maps/213/moscow/?ll=37.722565%2C55.806662&masstransit%5BstopId%5D=stop__9642962&mode=masstransit&z=16.52
+where stop id is: **9642962**
+
+if you want to look only specific routes, you can add them in routes section.
```

Comment:
If you want to track only specific routes, you can add them in the routes section.

Reply:
fixed